kong/plugins/oidc/utils.lua

```diff
@@ -67,6 +67,14 @@ function M.exit(httpStatusCode, message, ngxCode)
   ngx.exit(ngxCode)
 end
 
+function M.injectAccessToken(accessToken)
+  ngx.req.set_header("X-Access-Token", accessToken)
+end
+
+function M.injectIDToken(idToken)
+  ngx.req.set_header("X-ID-Token", idToken)
```

Comment:
The `idToken` is a Lua table, not a string. You need to handle it like the userinfo field (encode to a string and then to base64).
```suggestion
  local tokenStr = cjson.encode(idToken)
  ngx.req.set_header("X-ID-Token", ngx.encode_base64(tokenStr))
```